appcenter/crashes.py

```diff
@@ -351,6 +351,8 @@ def commit_symbol_upload(
 
         response = self.patch(request_url, data=data)
 
+        self.log.info(f"This is the current response: {response.json()}")
```

Comment:
We are running into the error `Expecting value: line 1 column 1 (char 0)` this may indicate we are not getting a json, it could probably be a 404 html. If the response is not a valid json, the `json()` would raise an exception and we may not get any useful information from the log.

Reply:
```suggestion
        self.log.info(f"committed symbol upload, upload id: {upload_id}, response: {response.status_code} {response.content}")
```